bimg/src/image.cpp

```diff
@@ -4831,35 +4905,91 @@ namespace bimg
 			break;
 
 		case TextureFormat::ASTC4x4:
+		case TextureFormat::ASTC5x4:
 		case TextureFormat::ASTC5x5:
+		case TextureFormat::ASTC6x5:
 		case TextureFormat::ASTC6x6:
 		case TextureFormat::ASTC8x5:
 		case TextureFormat::ASTC8x6:
+		case TextureFormat::ASTC8x8:
 		case TextureFormat::ASTC10x5:
-		    /*
+		case TextureFormat::ASTC10x6:
+		case TextureFormat::ASTC10x8:
+		case TextureFormat::ASTC10x10:
+		case TextureFormat::ASTC12x10:
+		case TextureFormat::ASTC12x12:
 			if (BX_ENABLED(BIMG_DECODE_ASTC) )
```

Comment:
I have a hack here to remove ASTC because it adds a bunch of extra code that is really only used for mobile. So the comment comment markers has to be re-added.